### PR TITLE
Fixes #36 (Client fails to retrieve items larger than 64 KB)

### LINKED ISSUE #36: Client fails to retrieve items larger than 64 KB
When attempting to retrieve items larger than 64 KB from the cache, the client fails with a timeout error. The set and insert operations work without issues, but the problem occurs specifically with the get and gat methods. This issue persists regardless of whether streaming is enabled or not.

Repro:

```
const value = Buffer.alloc(2 * 1024 * 1024, 'A'.charCodeAt(0));
await cache.set('key1', value);
const result = await cache.get('key1');
assert.deepStrictEqual(result, value);
```

### PR DESCRIPTION
**Pull request by unbornchikken**
Fix for https://github.com/alexzel/iomem/issues/36

### CODE REVIEW

**Review by alexzel — COMMENTED** [review]
Thanks for the contribution. Please see my comments. Also, would be good to add the corresponding test in `client.test.js`, but to make it work we need to increase max item size in `Dockerfile`:
```diff
-CMD echo $SASL_PASSWORD | saslpasswd2 -c memcached -p && exec memcached -S -u root
+CMD echo $SASL_PASSWORD | saslpasswd2 -c memcached -p && exec memcached -I 10m -S -u root
```

File: src/net.js [hunk 1]
```diff
@@ -122,7 +122,8 @@ class NetStream extends Transform {
         while (chunks.length >= HEADER_LENGTH && !error) {
           const header = parseHeader(chunks.slice(0, HEADER_LENGTH))
           if (header[0] !== RESPONSE_MAGIC) { // wrong magic
-            chunks = Buffer.alloc(0)
+            chunks = chunks.slice(HEADER_LENGTH)
+            continue
```
[the remark this answers is not in the document]
Comment (alexzel) [reply src/net.js:126]: Hmm.. or maybe just keep it as is.

[the remark this answers is not in the document]
Comment (unbornchikken) [reply src/net.js:126]: Yup, when the magic is not response, then it is a chunk of header data. I think it should be as is.

**Comment by unbornchikken** [comment]
Repro unit test added.

**Review by alexzel — APPROVED** [review]

**Comment by alexzel** [comment]
Please see the changes in v1.5.11

**Pull request merged**